fenapack/assembling.py

```diff
@@ -146,3 +166,35 @@ def pcd_bcs(self):
         except (AttributeError, AssertionError):
             raise AttributeError("BCs requested by PCD not available")
         return self._bcs_pcd
+
+
+class PCDForm(object):
+    """Wrapper for PCD operators represented by :py:class:`Form`.
```

Comment:
:py:class:`Form` is ambiguous

Reply:
:py:class:Form --> :py:class:`dolfin.Form` or :py:class:`ufl.Form`
Is this your point?